cli/src/electron/add.ts

```diff
@@ -1,11 +1,18 @@
 import { exec } from 'child_process';
 import { Config } from '../config';
 import { copyTemplate, hasYarn, installDeps, runTask } from '../common';
+import { readFileSync, writeFileSync } from '../util/fs';
+import * as path from 'path';
 
 export async function addElectron(config: Config) {
 
   await runTask(`Adding Electron project in: ${config.electron.platformDir}`, async () => {
-    return copyTemplate(config.electron.assets.templateDir, config.electron.platformDir);
+    const copyReturn = await copyTemplate(config.electron.assets.templateDir, config.electron.platformDir);
+    const capConfigName = JSON.parse(readFileSync(path.join(config.electron.platformDir, '../capacitor.config.json')) + '')['appName'];
+    const packageJSONParse = JSON.parse(readFileSync(path.join(config.electron.platformDir, './package.json')) + '');
```

Comment:
Same here, use require ad do't use `./`on the json name
```suggestion
    const packageJSONParse = require(join(config.electron.platformDir, 'package.json'));
```